routes/variableurl.go

```diff
@@ -0,0 +1,123 @@
+package routes
```

Comment:
You should comment your package.

Reply:
My bad, disregard that one, I thought you were making a new package for a moment.